### PR TITLE
Iht 4989 Pre-submission - Estate overview summary displayed based on different assets

### PR DESCRIPTION
**Pull request by grantrobson**

### CODE REVIEW

File: conf/pdf/templates/common/styles.xsl [hunk 1]
```diff
@@ -97,7 +104,17 @@
     <xsl:attribute name="font-size">12pt</xsl:attribute>
 </xsl:attribute-set>
 
-<xsl:attribute-set name="declaration">
+<xsl:attribute-set name="cell--heavy">
+    <xsl:attribute name="padding-top">8px</xsl:attribute>
```
Comment (adamliptrot-oc) [inline conf/pdf/templates/common/styles.xsl]: This should just extend the cell style. So cell--heavy should only contain the styles required to make it different (the two font styles).

File: conf/pdf/templates/common/table-row.xsl [hunk 2]
```diff
@@ -59,66 +59,110 @@
...
+                </fo:block>
+            </fo:table-cell>
+        </fo:table-row>
+    </xsl:template>
+
+    <xsl:template name="table-row--currency-right--heavy">
+        <xsl:param name="label"/>
+        <xsl:param name="value"/>
+        <fo:table-row xsl:use-attribute-sets="row">
+            <fo:table-cell xsl:use-attribute-sets="cell--heavy">
```
Comment (adamliptrot-oc) [inline conf/pdf/templates/common/table-row.xsl]: This should use the base cell styling and extend it with the cell--heavy style

Comment (grantrobson) [inline conf/pdf/templates/common/table-row.xsl]: Done

File: conf/pdf/templates/common/styles.xsl (hunk 1 above)
Comment (grantrobson) [inline conf/pdf/templates/common/styles.xsl]: Done

File: conf/pdf/templates/presubmission/estate-summary.xsl [hunk 3]
```diff
@@ -29,29 +29,77 @@
                                         select="scala:getMessagesText($translator, 'iht.estateReport.gifts.givenAway.title')"/>
                         <xsl:with-param name="value" select="$giftsTotal"/>
                     </xsl:call-template>
-                    <xsl:call-template name="table-row--currency-right">
+                    <xsl:if test="$estateOverviewDisplayMode=5">
```
Comment (tyagikumarvineet) [inline conf/pdf/templates/presubmission/estate-summary.xsl]: Better to add comments to identify the display modes in a more meaningful way

Comment (grantrobson) [inline conf/pdf/templates/presubmission/estate-summary.xsl]: Done

**Pull request merged**